extensions/awell/v1/actions/isPatientEnrolledInCareFlow/isPatientEnrolledInCareFlow.test.ts

```diff
@@ -109,13 +109,15 @@ describe('Is patient already enrolled in care flow action', () => {
           pathway_definition_id: 'pathway-definition-1',
           release_id: 'release-1',
           status: PathwayStatus.Active,
+          start_date: '2024-01-01',
```

Comment:
Are we sure we return just dates here and not ISO8601 datetime strings? Maybe it has no impact on the result of the test, but I'd make sure we align our test data with the data format the API returns.